mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -539,7 +560,7 @@
         <Period value="0.63"/>
     </CBehaviorCreepSource>
     <CBehaviorCreepSource id="makeCreep8x6">
-        <Period value="0.42"/>
+        <Period value="0.35"/>
```

Comment:
Hatchery/Lair/Hive: Creep spread interval decreased from 0.3 to 0.25 (from 0.42 to 0.35 on normal speed)

=> A smaller period means that they spread creep more often!

How do these values behave when they are not a multiple of 0.0625? I guess at some point it takes a game update frame less, so the period is sometime shorter (similar to persistent effects' and behaviors' periodic effects)

Reply:
I am testing a few different values that are close which do not run into this issue.

For example, 0.34 does not have that issue